tools/dgeni/processors/docs-private-filter.js

```diff
@@ -28,13 +28,27 @@ const INTERNAL_METHODS = [
 
 module.exports = function docsPrivateFilter() {
   return {
-    $runBefore: ['docs-processed'],
-    $process: function(docs) {
-      return docs.filter(d => !(hasDocsPrivateTag(d) || INTERNAL_METHODS.includes(d.name)));
-    }
+    $runBefore: ['categorizer'],
+    $process: docs => docs.filter(doc => validateDocEntry(doc))
   };
 };
 
+function validateDocEntry(doc) {
+  if (hasDocsPrivateTag(doc)) {
+    return false;
+  } else if (doc.docType === 'member') {
+    return validateMemberDoc(doc);
+  } else if (doc.docType === 'class') {
+    doc.members = doc.members.filter(memberDoc => validateMemberDoc(memberDoc));
```

Comment:
The docs of the class-members might have been removed already, but they are still referenced from the `class-doc`.